mass-deployer/pkg/mass-deployer/deployer.go

```diff
@@ -0,0 +1,206 @@
+package deployer
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"sync"
+
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/deployer"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/workloads"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-proxy/pkg/types"
+	"github.com/threefoldtech/tfgrid-sdk-go/mass-deployer/internal/parser"
+	"github.com/threefoldtech/zos/pkg/gridtypes"
+)
+
+type Deployer struct {
+	TFPluginClient deployer.TFPluginClient
+}
+
+type vmDeploymentInfo struct {
+	nodeID         uint32
+	vmName         string
+	deploymentName string
+}
+
+type VMInfo struct {
+	Name      string
+	PublicIP4 string
+	PublicIP6 string
+	YggIP     string
+	IP        string
+	Mounts    []workloads.Mount
+}
+
+func NewDeployer(conf parser.Config) (Deployer, error) {
+	network := conf.Network
+	log.Printf("network: %s", network)
```

Comment:
use zerolog

Reply:
this is zerolog, not sure what you mean here